### PR TITLE
Replaced Kerbalstuff by Github

### PR DESCRIPTION
**Pull request by sswelm**
since, KerbalStuff has died, it needs to update from somewhere else

### CODE REVIEW

File: NetKAN/KSPInterstellarExtended.netkan [hunk 1]
```diff
@@ -1,7 +1,10 @@
 {
     "spec_version"       : 1,
     "identifier"         : "KSPInterstellarExtended",
-    "$kref"              : "#/ckan/kerbalstuff/791",
+    "$kref"              : "#/ckan/github/sswelm/KSPInterstellar",
+    "$vref" 		 : "#/ckan/ksp-avc"
```
Comment (pjf) [inline NetKAN/KSPInterstellarExtended.netkan]: Oops, we're missing a comma at the end of this line! Can we add one in?

**Comment by pjf** [comment]
Thank you! I think we'll see a lot of these over the next few days.

We seem to be missing a comma on line 5, any chance you can add it in?

**Comment by dbent** [comment]
You also want to add a `resources.homepage` as that will no longer be automatically populated.

**Comment by sswelm** [comment]
I added the comma and made a new pull request

**Pull request closed without merging**